pkg/apis/serving/v1beta1/service_validation_test.go

```diff
@@ -603,3 +605,120 @@ func TestServiceSubresourceUpdate(t *testing.T) {
 		})
 	}
 }
+
+func getServiceSpec(image string) ServiceSpec {
+	return ServiceSpec{
+		ConfigurationSpec: ConfigurationSpec{
+			Template: RevisionTemplateSpec{
+				Spec: RevisionSpec{
+					PodSpec: corev1.PodSpec{
+						Containers: []corev1.Container{{
+							Image: image,
+						}},
+					},
+					TimeoutSeconds: ptr.Int64(config.DefaultMaxRevisionTimeoutSeconds),
+				},
+			},
+		},
+		RouteSpec: RouteSpec{
+			Traffic: []TrafficTarget{{
+				LatestRevision: ptr.Bool(true),
+				Percent:        100,
+			}},
+		},
+	}
+}
+
+func TestServiceAnnotationUpdate(t *testing.T) {
+	const (
+		u1 = "oveja@knative.dev"
+		u2 = "cabra@knative.dev"
+		u3 = "vaca@knative.dev"
+	)
+	tests := []struct {
+		name string
+		prev *Service
+		this *Service
+		want *apis.FieldError
+	}{{
+		name: "update creator annotation",
+		this: &Service{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "valid",
```

Comment:
I think the name "valid" might be misleading here, since we are expecting an error

Reply:
In this case we are validating annotation not name 
same like https://github.com/knative/serving/blob/master/pkg/apis/serving/v1beta1/service_validation_test.go#L132 
so i feel name` valid` is not wrong WDYT?

Reply:
Ah I see. That makes sense. Thanks for the clarification!